app/routines/admin/search_security_log.rb

```diff
@@ -64,26 +64,32 @@ def self.sanitize_times(time_strings, now)
 
     def exec(params = {}, options = {})
 
+      sec = SecurityLog.arel_table
+      usr = User.arel_table
+      ap = Doorkeeper::Application.arel_table
+
```

Comment:
I'm not a fan of really short variable names unless they are used nearby. Not a showstopper, but it is always clearer to be more descriptive. Just a suggestion.